app/models/book.rb

```diff
@@ -1,8 +1,27 @@
+#
+# frozen_string_literal: true
+
 class Book < ApplicationRecord
   has_many :publications
   has_many :authors, through: :publications, source: :author
   has_many :comments, as: :commentable
-  validates :title, presence: true
+  validates :title, presence: true, format: { with: /\A[a-zA-Z]+\z/,
```

Comment:
please, format this in a more readable way.

somthing like:
validates(
:name,
etc.
)